### PR TITLE
Fix push to csv Github action

### PR DESCRIPTION
**Pull request by annoyingpr**

### CODE REVIEW

**Comment by rahularoradfs** [comment]
@annoyingpr @simonarocco do we have a mechanism in place yet such that this data updates daily? If not, what's the plan to regularly update the data manually, and to get that mechanism in place soon?

File: .github/workflows/push-csv-to-repo.yml [hunk 1]
```diff
@@ -3,43 +3,50 @@ name: Push CSV to Repo
...
+        run: python3 $GITHUB_WORKSPACE/iit-backend/app/github_public_repo/estimate_csv_creator.py
+        env:
+          AIRTABLE_API_KEY: ${{ secrets.AIRTABLE_API_KEY }}
+          AIRTABLE_BASE_ID: ${{ secrets.AIRTABLE_BASE_ID }}
+          FLASK_ENV: 'dev'
+          LOG_CONFIG_PATH: 'iit-backend/logging.cfg'
+          LOG_FILE_PATH: 'iit-backend/logfile.log'
+          PYTHONPATH: 'iit-backend'
+    
+      - name: Copies CSV to sars-cov-2-data
```
Comment (simonarocco) [inline .github/workflows/push-csv-to-repo.yml:39]: This step just copies the CSV to the iit backend repo -- where is the step that actually pushes this to the main branch? Or does this do this automatically on the main branch?

**Review by simonarocco — APPROVED** [review]
lgtm -- just had one question for you. as long as this has been tested (make sure you test that it runs at 12 UTC) i'm fine to merge it

File: .github/workflows/push-csv-to-repo.yml (hunk 1 above)
Comment (atatmaja) [reply .github/workflows/push-csv-to-repo.yml:39]: Based on the action docs (https://github.com/marketplace/actions/push-a-file-to-another-repository), seems like it actually does push the file as well.

@annoyingpr can you update the step name to be clear that both copying and pushing are happening here? Thanks

Comment (annoyingpr) [reply .github/workflows/push-csv-to-repo.yml:39]: yeah just realized that the pushing part wasn't as clear, will update the step name!

**Pull request merged**